package.json

```diff
@@ -6,8 +6,9 @@
   "ungitPluginApiVersion": "0.2.0",
   "scripts": {
     "start": "node ./bin/ungit",
-    "test": "grunt test",
+    "test": "prettier --check . && grunt test",
```

Comment:
Currently all tooling is run through grunt and while I think it might be good to switch to npm scripts in general, that might be a different change.

Reply:
my grunt is rusty, any quick hints for how to achieve this?

Reply:
I would recommend looking at `Gruntfile.js` and [grunt-prettier](https://www.npmjs.com/package/grunt-prettier) package

Reply:
@jung-kim I ended up doing it in plain grunt, would the grunt-prettier package add anything?